src/litserve/loggers.py

```diff
@@ -112,7 +112,10 @@ def _process_logger_queue(loggers: List[Logger], queue):
         while True:
             key, value = queue.get()
             for logger in loggers:
-                logger.process(key, value)
+                try:
+                    logger.process(key, value)
+                except Exception as e:
+                    module_logger.error(f"Error processing log entry with key {key} and value {value}: {e}")
```

Comment:
Since there might be multiple loggers, it would be useful to also log exactly what logger failed to log.

Reply:
I agree

Reply:
thanks for feedback @Andrei-Aksionov @lantiga! Would logging the exception `logger.error(f"... {e}")` show the error from exactly which logger it failed?

Reply:
Added `logger.__class__.__name__` in the error msg